src/components/testCaseList/TestCase.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import tw from "twin.macro";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowRight, faArrowUp } from "@fortawesome/free-solid-svg-icons";
+import { useNavigate } from "react-router-dom";
+import { Button } from "@madie/madie-components";
+
+const TH = tw.th`p-3 border-b text-left text-sm font-bold uppercase`;
+const TD = tw.td`p-3 whitespace-nowrap text-sm font-medium text-gray-900`;
+const EditButton = tw.button`text-blue-600 hover:text-blue-900`;
+
+const TestCase = (props) => {
+  const navigate = useNavigate();
+  const expanded = props.activeItem === props.index;
+  const cls = expanded ? "item-active" : "";
+  const testCase = props.testCase;
+  const isExecuteButtonClicked = props.isExecuteButtonClicked;
+
+  const toggleExpandedHandler = (index) => {
+    props.setActiveItem(index);
+  };
+
+  return (
+    <tr
+      tw="border-b"
+      key={props.testCase.id}
+      data-testid={`test-case-row-${props.testCase.id}`}
```

Comment:
same here `testCase.id`

Reply:
This was resolved so I'll mark it as Resolved.